src/spec/ColorSpec.ts

```diff
@@ -17,6 +17,18 @@ describe('A color', () => {
       expect(color.toString()).toBe('rgba(0, 0, 0, 1)');
    });
 
+   it('should display hex values #000000', () => {
+      expect(color.toString('hex')).toBe('#000000');
+   });
+
+   it('should display hsla(0,0,0,1)', () => {
+      expect(color.toString('hsl')).toBe('hsla(0, 0, 0, 1)');
+   });
+
+   it('should display an error message', () => {
+      expect(color.toString('invalid')).toThrow(new Error('Invalid Color format'));
```

Comment:
Once you add type unions this will be a compiler error, you can force ignore it by doing `'invalid' as any` I think.